src/components/organisms/modals/TooltipModal/classes.module.scss

```diff
@@ -10,6 +10,10 @@
 .contentStyle {
```

Comment:
**[blocker]**

Seems that the list ( ul/li tags ) styling we currently have here doesn't match the design.

1. One thing we should do here is create some mixin (under styles/mixins) for the styling we want to apply to these helper html sections. This will not use any classNames, since the content we get will be pure html, so it will just have styles for html tags. This way we can include the exact same styles for both the manual page and this modal.

2. Once the styles in 1. are created, they can be included under the .contentStyle class in the TooltipModal.

3. On the manual page we should also wrap the {manualContent} with some div and give a className to that div, so we can add the same styles there with the mixin.

4. For now you can just check some of the example we have in design for the styles that should be applied. Mostly everything else seems fine, just the ul/li part might need some work. Later there might be new tags used in the manual page and we might need to edit those styles (This is why it would be good to have a mixin for it that both the tooltip and manual page use)

<img width="734" alt="Screenshot 2023-09-06 at 10 09 06" src="https://github.com/keeleinstituut/tv-web/assets/26221722/f837952e-82de-443b-8059-aebb69b4ee1f">